### ISSUE TITLE
[TW-1251] Consider optional integration of readline

### ISSUE THREAD

**Issue by taskwarrior** [GothenburgBitFactory/taskwarrior#1285]
_Paul Beckingham on 2009-11-26T09:56:20Z says:_ 

 Integrating readline (optionally) would improve the task shell for all users.
See http://taskwarrior.org/issues/show/323.

We should consider integrating readline, and assess the difficulty and appropriateness.

**Comment by taskwarrior** [comment]
Migrated metadata:
```
Created: 2009-11-26T09:56:20Z
Modified: 2014-02-09T16:59:16Z
```

**Comment by taskwarrior** [comment]
_Paul Beckingham on 2010-02-10T17:53:30Z says:_ 

 Readline support is trivial to add to the code - it's more of a packaging effort.

**Comment by taskwarrior** [comment]
_Paul Beckingham on 2011-01-23T01:06:11Z says:_ 

 Bumped to 1.9.5 for the same reasons as #328.

**Comment by taskwarrior** [comment]
_Paul Beckingham on 2014-02-09T16:59:16Z says:_ 

 Fixed in 2.3.0
